packages/configs/README.md

```diff
@@ -0,0 +1,16 @@
+# @pantheon-systems/configs
+
+ESLint configs for `@pantheon-systems` npm packages.
+
+The configs are currently unpublished.
+
+## How To Add A New ESLint config
+
+1. Add a \*.js file to the root of this directory with the name of the config,
+   for example `vue.js`
+1. Extend the base plugin from your plugin unless it is meant to be standalone
+1. If a new parser is used, add it to the `dependencies` in the `package.json`
+1. If new plugins are used, add them to the `peerDependencies` in the
+   `package.json`
+1. From your .js file, export your config
+1. Add the config to the `files` array in the `package.json`
```

Comment:
This is probably still fine but ensure the accuracy here. We might want to move this to CONTRIBUTING or keep it in the local one? Unsure of how detailed this readme needs to be at this point.

Reply:
After thinking about it more I feel like making this README light and keeping a more in depth one with contributing details in our local workspace makes sense. Just pushed an update to reflect this.